CutestTorture/Assets/Scripts/Player/CoinPickup.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
+
+namespace Player
+{
+    public class CoinPickup : MonoBehaviour
+    {
+        private int _nCoins;
```

Comment:
`_nCoins` -> `_numberOfCoins` or just `_coins`
There is no need to shorten the names